cobra/core/model.py

```diff
@@ -778,6 +778,43 @@ def to_array_based_model(self, deepcopy_model=False, **kwargs):
         from cobra.core.arraybasedmodel import ArrayBasedModel
         return ArrayBasedModel(self, deepcopy_model=deepcopy_model, **kwargs)
 
+    def slim_optimize(self, error_value=0., message=None):
```

Comment:
I realised this and change it to `error_value=float('nan')` which I think should be less confusing. Personally I like to option get a value if error to avoid the long try/except blocks.  `raise_error=False` as you propose is also an option but then don't find it obvious what the return value should be..